kolibri/utils/cli.py

```diff
@@ -333,6 +357,29 @@ def initialize(skip_update=False):
         # our Kolibri version based update logic.
         run_plugin_updates()
 
+    check_content_directory_exists_and_writable()
+
+    if not skip_update:
```

Comment:
I'm wondering if this code block can be combined with the one above here: https://github.com/learningequality/kolibri/pull/6580/files#diff-8f9ca278df35a8b43572c4b10dafe306R355 as they have the same condition. Thank you!

Reply:
That could make a lot of sense for readability, thanks!

Reply:
To reduce indentation and make the function easier to understand, I usually prefer to "bail early" rather than nesting logic inside a conditionals. Concretely:

```python
def initialize(skip_update=False):

    # ...always do this stuff...

    if skip_update:
        return

    # ...conditionally do this stuff...
```

Reply:
@indirectlylit that's a good structure :+1: it's a bit hard to make the `initialize()` function prettier - there are several of the checks and upgrade behaviors where the order and conditions seem sensitive. I can't tell for sure though, it requires a lot of pondering for each line.

I've gathered it a bit and moved the content directory check now.

Reply:
I hope the change is okay now, if I clean up `initalize()` more, I fear introducing regressions in the regression test PR :)